x-pack/plugins/synthetics/public/apps/synthetics/routes.tsx

```diff
@@ -86,9 +92,29 @@ const getRoutes = (
       component: () => <MonitorDetailsPage />,
       dataTestSubj: 'syntheticsMonitorDetailsPage',
       pageHeader: {
-        children: <MonitorDetailsPageHeader />,
         pageTitle: <MonitorDetailsPageTitle />,
```

Comment:
I'm not able to see confirm this, but please check that the page title is using `EuiTitle size="m"`. By default it uses `l`.

FYI - the "root" pages for Synthetics use a large title, whereas all the inner pages, like the monitor detail page, use a medium title. This is to help account for longer title names.

Reply:
I don't think I can easily change this. The templates use [`EuiPageHeader`](https://elastic.github.io/eui/#/layout/page-header) for consistency. The component accepts a `pageTitleProps` to adjust certain aspects of the underlying title component but for some reason `size` is not one of them.

@elastic/eui-design could you shed some light on the reasoning behind this decision? Could it be possible to adjust it in some other way?

Reply:
@miukimiu may be able to answer more thoroughly but my personal guess is consistency. It really doesn't make sense to have page title sizes jumping all over the place when users navigate from one app to another - keeping the same size for the page title throughout multiple pages makes sense.

Reply:
If it's not possible with the page template, don't worry about it. Though, I want to wait for @miukmiu to chime in :) A "jump" from large to medium doesn't seem like a big issue to me, especially for these pages where the titles can be user generated content.

Reply:
@afgomez I checked the docs and it shows `size` as valid a prop. Perhaps there is an EUI bug or typo? 
![image](https://user-images.githubusercontent.com/847805/194124430-5e97a738-39db-4e17-bbf9-41709495d861.png)https://elastic.github.io/eui/#/layout/page-header

Reply:
`Omit<>` means that the `size` prop is **not** allowed 😅 apologies for the confusion, our types docgen is occasionally not super helpful especially for those unfamiliar with Typescript.

Reply:
I'll leave it with the current size for now then.